grails-app/controllers/au/org/ala/ecodata/ProjectActivityController.groovy

```diff
@@ -69,4 +71,29 @@ class ProjectActivityController {
         }
     }
 
+    /**
+     * Soft delete project activity (no cascading - to prevent bulk activity and records update)
+     * @param id - project activity id
+     *
+     * @return
+     */
+    @RequireApiKey
+    def delete(String id) {
```

Comment:
Would you mind removing the delete method from here and from ProjectActivityService to avoid a clash with #132?